.github/ubiquibot-config.yml

```diff
@@ -31,6 +31,7 @@ default-labels:
   - "Time: <1 Hour"
   - "Priority: 0 (Normal)"
 payment-permit-max-price: 1000
+private-key-encrypted: ""
```

Comment:
I think this is not needed here. `private-key-encrypted` is an org config. This file is repo config. This property is never read from repo config.

Reply:
So, How should i go about the QA

Reply:
1. Make an org 
2. Add your bot to your org
3. Don't add private key to config

Reply:
in the build step in `build.yml` SupaBase access key is required. So, I made a new project added the API key to the field it still gave error then I tried adding the jwt secret it stilll gave error. what should I enter in that field?

Reply:
These docs are helpful https://github.com/ubiquity/ubiquibot/blob/development/supabase/README.md
What errors are you getting?

Reply:
I got an error which said the format of my key is wrong.
where should I find my the key
Is it the API key key or jwt token

Reply:
Look into readme file in PR https://github.com/ubiquity/ubiquibot/pull/607 , and adds info about **Payment Permits in a local instance**. It shows you how to generate a private key and use it. This will be helpful in fixing this issue and creating a QA. Please note there are two different kinds of private keys, one is used to encrypt another private key.

Reply:
> These docs are helpful https://github.com/ubiquity/ubiquibot/blob/development/supabase/README.md What errors are you getting?

SUPABASE_KEY=XXX
The error said the format of my SUPABASE_KEY is incorrect where should I find the Key value? It could be that I put the wrong key in.

Reply:
This is the correct key

![image](https://github.com/ubiquity/ubiquibot/assets/139262667/0625e591-9e5c-466a-bfc0-8253cecec764)

Reply:
It's working now but the format of `SUPABASE_ACCESS_TOKEN` seems to be wrong where may I find that?

Reply:
In case you haven't found it, it's [here](https://supabase.com/dashboard/account/tokens)

Reply:
now the bot is running but 
![ubi](https://github.com/ubiquity/ubiquibot/assets/119438857/db8ddcd8-fd43-4e35-8055-81d02ca0977e)
its giving the same message for every command

Reply:
@wannacfuture do you know the solution?

Reply:
@Sadaf-A 
Would you gimme the link, not screenshot?
Let me check it.

Reply:
> @Sadaf-A 
> Would you gimme the link, not screenshot?
> Let me check it.
> 
> And also you should check the config file if it has been set correctly like the file in ubiquibot staging repo(e.g.);

https://github.com/testing-ubiquibot1/ubiquibot/issues/1 here is the link
thanks for offering help!